packages/components/components/elvis-datepicker/src/react/dateHelpers.ts

```diff
@@ -27,36 +29,42 @@ export const dateIsWithinMinMaxBoundary = (
   return !dateIsBeforeMinDate && !dateIsAfterMaxDate;
 };
 
-export const isSameDay = (d1?: Date | null, d2?: Date | null): boolean => {
+export const isSameDay = (lang: LanguageCode, d1?: Date | null, d2?: Date | null): boolean => {
   if (!d1 || !d2) {
     return false;
   }
 
   return (
-    formatDate(d1, { year: 'numeric', month: 'numeric', day: 'numeric' }) ===
-    formatDate(d2, { year: 'numeric', month: 'numeric', day: 'numeric' })
+    formatDate(lang, d1, { year: 'numeric', month: 'numeric', day: 'numeric' }) ===
+    formatDate(lang, d2, { year: 'numeric', month: 'numeric', day: 'numeric' })
   );
 };
 
-export const getDayName = (date: Date): string => {
-  return date.toLocaleDateString('nb-NO', { weekday: 'short' }).substring(0, 2);
+export const getDayName = (lang: LanguageCode, date: Date): string => {
+  return lang === 'no'
+    ? date.toLocaleDateString('nb-NO', { weekday: 'short' }).substring(0, 2)
+    : date.toLocaleDateString('en-GB', { weekday: 'short' }).substring(0, 2);
 };
```

Comment:
I suggest checking the language inside the `toLocaleDateString` method, to reduce the amount of repeated code. Something like this:
```ts
date.toLocaleDateString(lang === 'no' ? 'nb-NO' : 'en-GB', { weekday: 'short' }).substring(0, 2);
```